src/app/accounts/active-accounts.ts

```diff
@@ -34,7 +34,7 @@ export const getRecentlyActiveAccounts = async function* ():
     const wallets = await WalletsRepository().listByAccountId(account.id)
     if (wallets instanceof Error) {
       recordExceptionInCurrentSpan({
-        error: "impossible to listByAccountId",
+        error: wallets,
```

Comment:
What was the motivation for switching out some of the strings with the underlying errors?

Reply:
I think was a mistake.. otel include string as a possible value for Exception but that is not our intention. We need a Domain error to do tracing properly